test/suites/smoke/test-polkadot-decoding.ts

```diff
@@ -62,6 +64,28 @@ describeSuite({
       title: "should be decodable",
       timeout: ONE_HOURS,
       test: async function () {
+        // Test case reproduction
+        if (STARTING_KEY_OVERRIDE) {
```

Comment:
This is an empty string `const`, which is a falsy value and will always evaluate to false